crates/kubelet/src/kubelet.rs

```diff
@@ -125,12 +126,9 @@ mod test {
     use std::collections::BTreeMap;
 
     fn mock_client() -> kube::Client {
-        kube::config::Configuration {
-            base_path: ".".to_string(),
-            client: reqwest::Client::new(),
-            default_ns: " ".to_string(),
-        }
-        .into()
+        kube::Client::new(kube::Config::new(
+            reqwest::Url::parse("http://example.com").unwrap(),
```

Comment:
slight nitpick: even though this is a mock test, in order to absolutely prevent any chance of reaching that URL, can we change this to `http://127.0.0.1:8080`, or is that not possible?